build/yaml/crd/vpc/crd.nsx.vmware.com_ipblocksinfos.yaml

```diff
@@ -48,7 +48,7 @@ spec:
             description: |-
               PrivateTGWIPCIDRs is a list of CIDR strings. Each CIDR is a contiguous IP address
               spaces represented by network address and prefix length. The visibility of the
-              IPBlocks is PrivateTWG. Only IPBlocks in default project will be included.
+              IPBlocks is Private Transit Gateway. Only IPBlocks in default project will be included.
```

Comment:
I think it's more about the typo, use `PrivateTGW` as it's the type in NSX API

Reply:
Updated, thanks!

Reply:
No it should be plain English in the description ideally.